Documentation/Manual/Texinfo/extensions.txi

```diff
@@ -27,6 +28,75 @@ the manual. Drei documentation is provided as a separate document.
 @node Text field gadget
 @node Text editor pane
 
+@node Extended text formatting
+@section Extended text formatting
+
+@code{standard-extended-output-stream} has been extended by protocols
+complementary to already defined ones. Some were clearly missing given
+how it is specified - like margins and word wrap. Some are useful in
+contemporary text displaying applications like text direction and
+alignment.
+
+@subsection Page abstraction
+
+@anchor{Function clime:page-initial-position}
+@anchor{Function clime:page-final-position}
+@deffn {Function} {clime:page-initial-position} stream
+@deffnx {Function} {clime:page-initial-position} stream
+@findex clime:page-initial-position
+@findex clime:page-final-position
+Both functions return two values, x and y coordinates of the
+corresponding position. Initial position is where the cursor is placed
+on a fresh page, and the final position is where the cursor is placed
+right before the page ends. Coordinates depend on current margins and
+text alignment.
+@end deffn
+
+@anchor{Function clime:stream-page-region}
+@deffn {Function} {clime:stream-page-region} stream
+@findex clime:stream-page-region
+Functions returns a region which corresponds to the stream page
+format. This region implies the stream margins.
```

Comment:
What does the region imply? Is this a typo, and the intended word is `includes`?

Reply:
wording is correct, margins are implied by the region (to be exact: its bounding rectangle).